### PR TITLE
Added CameraAction class

### PR DESCRIPTION
**Pull request by NeilMalcolm**
Added CameraAction class as well as added to IActionExtensions and Camera value added to ActionType as mentioned in issue #18.

### CODE REVIEW

File: src/LineBot/Actions/CameraAction.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Line
+{
+	/// <summary>
+	/// Encapsulates a template camera action.
```
Comment (dlemstra) [inline src/LineBot/Actions/CameraAction.cs]: The word template should be removed, I forgot to remove it from the other actions...

Comment (NeilMalcolm) [reply src/LineBot/Actions/CameraAction.cs]: Sure thing! I'll get that straightened out and add the unit tests 👍

Comment (dlemstra) [reply src/LineBot/Actions/CameraAction.cs]: Feel free to take a peek at the other unit tests for some ideas on how I set those up.

**Comment by NeilMalcolm** [comment]
Hey @dlemstra, think I've sorted out the issues you mentioned above. Added the UnitTests also.

**Comment by dlemstra** [comment]
Thanks @NeilMalcolm!

**Pull request merged**